ext/opcache/jit/zend_jit_vm_helpers.c

```diff
@@ -613,20 +613,36 @@ static int zend_jit_trace_subtrace(zend_jit_trace_rec *trace_buffer, int start,
  *  Trace Linking Rules
  *  ===================
  *
- *                                          flags
- *          +----------+----------+----------++----------+----------+----------+
- *          |                                ||              JIT               |
- *          +----------+----------+----------++----------+----------+----------+
- *   start  |   LOOP   |  ENTER   |  RETURN  ||   LOOP   |  ENTER   |  RETURN  |
- * +========+==========+==========+==========++==========+==========+==========+
- * | LOOP   |   loop   |          | loop-ret || COMPILED |   LINK   |   LINK   |
- * +--------+----------+----------+----------++----------+----------+----------+
- * | ENTER  |INNER_LOOP| rec-call |  return  ||   LINK   |   LINK   |   LINK   |
- * +--------+----------+----------+----------++----------+----------+----------+
- * | RETURN |INNER_LOOP|          |  rec-ret ||   LINK   |          |   LINK   |
- * +--------+----------+----------+----------++----------+----------+----------+
- * | SIDE   |  unroll  |          | side-ret ||   LINK   |   LINK   |   LINK   |
- * +--------+----------+----------+----------++----------+----------+----------+
+ *                         opline trace flags
+ *                  +----------+----------+----------+
+ *                  |   LOOP   |  ENTER   |  RETURN  |
+ *         +========+==========+==========+==========+
+ *         | LOOP   |   loop   |          | loop-ret |
+ *         +--------+----------+----------+----------+
+ *         | ENTER  |INNER_LOOP| rec-call |  return  |
+ *  start  +--------+----------+----------+----------+
+ *  flags  | RETURN |INNER_LOOP|          |  rec-ret |
+ *         +--------+----------+----------+----------+
+ *         | SIDE   |  unroll  |          | side-ret |
+ *         +--------+----------+----------+----------+
+ *
+ *
+ *                         opline trace flags
+ *                  +----------+----------+----------+
+ *                  |             JITED              |
```

Comment:
```suggestion
 *                         opline trace flags before JIT
 *                  +----------+----------+----------+
 *                  |   LOOP   |  ENTER   |  RETURN  |
 *         +========+==========+==========+==========+
 *         | LOOP   |   loop   |          | loop-ret |
 *         +--------+----------+----------+----------+
 *         | ENTER  |INNER_LOOP| rec-call |  return  |
 *  start  +--------+----------+----------+----------+
 *  flags  | RETURN |INNER_LOOP|          |  rec-ret |
 *         +--------+----------+----------+----------+
 *         | SIDE   |  unroll  |          | side-ret |
 *         +--------+----------+----------+----------+
 *
 *
 *                         opline trace flags after JIT
```

Reply:
I'm not sure this emphases on the right thing.

`JITED` is a flag (`ZEND_JIT_TRACE_JITED`), and this is meant to be read as "when the JITED flag is set in addition to one of LOOP, ENTER, or RETURN".

What do you think of this:

```
 *                         opline trace flags
 *                  +----------+----------+----------+
 *                  |    JITED (opline is JIT'ed)    |
 *                  +----------+----------+----------+
 *                  |   LOOP   |  ENTER   |  RETURN  |
```

Or maybe:

```
 * When opline it JIT'ed:
 *                         opline trace flags
 *                  +----------+----------+----------+
 *                  |   LOOP   |  ENTER   |  RETURN  |
```

Reply:
It took me a while to understand the headline difference. When the 1st/single headline is different, it is much quicker to read. That is all.